.github/action-common-python-release/action.yml

```diff
@@ -71,3 +71,9 @@ runs:
     with:
       name: wheels_${{ inputs.package-name }}-${{ inputs.python-version }}-${{ inputs.artifact-key }}
       path: dist/
+
+  - name: unzip test
+    shell: bash
+    run: |
+      unzip -l dist/*
+      unzip -T dist/*
```

Comment:
Aren't we already testing wheel installation? I figure that would cover testing the zip file as well.

Reply:
I'd think so. But it's still failing 😞

Reply:
removed, you are right.